utils/controleval.py

```diff
@@ -182,6 +182,18 @@ def create_implicit_control_lists(ctrls, control_list):
     return control_list
 
 
+def count_rules_and_vars_in_control(ctrl):
+    rules = [item for item in ctrl.rules if "=" not in item]
+    variables = [item for item in ctrl.rules if "=" in item]
+    return len(rules), len(variables)
+
+
+def count_rules_and_vars(ctrls):
+    rules_count = sum(count_rules_and_vars_in_control(ctrl)[0] for ctrl in ctrls)
+    vars_count = sum(count_rules_and_vars_in_control(ctrl)[1] for ctrl in ctrls)
```

Comment:
It would be better if `count_rules_and_vars_in_control` returned a namedtuple, it would allow you to avoid using the indices. Another possible approach would be to split `count_rules_and_vars` to  two separate functions `count_rules` and `count_vars`.

Reply:
Thanks for the thoughts @jan-cerny . I tried to make it a little more readable using namedtuple in the last commit. Could you take a look, please?